pipelines/h6c/rtp/v2/stage_1_task_scripts/do_ALL_SKY_IMAGE_SNAPSHOT.sh

```diff
@@ -0,0 +1,56 @@
+#! /bin/bash
+set -e
+
+
+#This script generates an all-sky image of the sky as a jpg
+# a subsequent job collects them all into an image
+
+
+# for now assumes scripts are to be found in ~/bin/
+
+src_dir="$(dirname "$0")"
+source ${src_dir}/_common.sh
+
+# Parameters are set in the configuration file. Here we define their positions,
+# which must be consistent with the config.
+fn=${1}
+
+# Export variables used by the notebook
+export SUM_FILE="$(cd "$(dirname "$fn")" && pwd)/$(basename "$fn")"
+#CALFILE=/lustre/aoc/projects/hera/H6C/2459853/zen.2459853.35006.sum.omni.calfits
+
+
+#take a raw HERA snapshot to widefield image and render it as a jpg
+mamba activate HERA2023
+#step 1 make a calibrated MS.  read raw, apply cal, write ms
+# comment --cal_file to LET CAL FLOAT
+python ~/bin/uvh5tocalibratedms.py ${SUM_FILE} # --cal_file ${CALFILE}
```

Comment:
Are we not calibrating the file? Won't that break things?

Also, don't love the hard-coded path here for `uvh5tocalibratedms.py`

Reply:
When you don't give it a cal file the script assumes the cal file exists in the same path as the raw data.  

In the interest of getting this rolled out quickly (to respond to what appears to be a rapidly worsening RFI situation) I did not make the scripts installable. Easiest way to deal is to put them in bin.

Reply:
I don't love all this hard-coding, so perhaps we can open an issue to get this working in a way that's easier to update and debug?

Reply:
issue #1 on hera_argus is to add a proper setuptools install.